staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,63 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, this directory and it's sub-directories will be skipped.
+- Adding `.import-restrictions` files does not add them to CI runs. In order for them to be added, they need to be explicitly added to `arguments.InputDirs`. `InputDirs` is defined within `import-boss/main.go`.
```

Comment:
> In order for them to be added, they need to be explicitly added to `arguments.InputDirs`.

->

> They need to be explicitly added to `arguments.InputDirs`.

Reply:
+1

Reply:
After the commit 558f6aab56c0e983741d7df6eb8ffc79a19a8672 we need to specify directories which contain `.import-restrictions` on hack/verify-import-boss.sh instead of import-boss/main.go.
So we need to update here.

/cc @oomichi

Reply:
@neolit123 I'm interpreting this comment as, replace the former with the latter?

Reply:
my comment is no longer valid. based on @oomichi 's comment i think we should formulate this in the lines of:

> - To add more paths to be checked by the k8s CI, add them in the file `hack/verify-import-boss.sh

its here:
https://github.com/kubernetes/kubernetes/blob/master/hack/verify-import-boss.sh#L33